server/routes/user.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const { check, validationResult } = require('express-validator');
+const router = express.Router();
+
+const User = require('../database/models/User');
```

Comment:
can we use a path name from the root of the folder as opposed to a relative path name here?